backend/worker/requirements.txt

```diff
@@ -9,6 +9,7 @@ joblib==1.2.0
 mitmproxy==9.0.1
 mitmproxy_wireguard==0.1.23
 numpy==1.24.2
+pandas
```

Comment:
@mbachtell-nais should we pin the pandas version here?

Reply:
we can

how about 1.5.1?

Reply:
Yes